engine/pkg/orm/util_test.go

```diff
@@ -14,13 +14,24 @@
 package orm
 
 import (
+	"context"
 	"errors"
+	"regexp"
 	"testing"
+	"time"
 
-	cerrors "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/DATA-DOG/go-sqlmock"
 	"github.com/stretchr/testify/require"
+
+	metaMock "github.com/pingcap/tiflow/engine/pkg/meta/mock"
+	cerrors "github.com/pingcap/tiflow/pkg/errors"
 )
 
+func init() {
+	// default log level is info, enable debug here for easy debug
+	// log.SetLevel(zapcore.DebugLevel)
+}
+
```

Comment:
Not used

Reply:
They may be used for debug in the future.
Or some people have no idea how to enable trace log for this test.